Source/missiles.h

```diff
@@ -132,6 +138,29 @@ struct Missile {
 	{
 		return _misource == -1;
 	}
+
+	[[nodiscard]] Player *SourcePlayer()

```

Comment:
Members are lower case (only global symbols are upper)
```suggestion
	[[nodiscard]] Player *sourcePlayer()
```